modules/tinymce/src/core/test/ts/browser/DragnDropCEFTest.ts

```diff
@@ -0,0 +1,62 @@
+import { Mouse, UiFinder, Waiter } from '@ephox/agar';
```

Comment:
Is there a reason this wasn't added to the existing `DragDropOverridesTest`? As this is just updating that and we already have a test for that does the same or a very similar thing.

- [ ] Reconsider using a new test for this when a test file already exists with similar tests

Reply:
you are right, I moved the tests to a new `context` in `DragDropOverridesTest`